package.json

```diff
@@ -35,7 +35,7 @@
     "eth-rpc-errors": "^4.0.3",
     "ethers": "^5.6.6",
     "fortmatic": "^2.3.0",
-    "preact": "https://github.com/preactjs/preact",
+    "preact": "^10.7.2",
```

Comment:
`preact` failed to install for me, so I reinstalled it, looks like they've merged the `svg` types PR :)
https://github.com/preactjs/preact

Reply:
It also failed the GH Actions